packages/@react-aria/gridlist/src/useGridList.ts

```diff
@@ -167,7 +168,7 @@ export function useGridList<T>(props: AriaGridListOptions<T>, state: ListState<T
   );
 
   if (isVirtualized) {
-    gridProps['aria-rowcount'] = state.collection.size;
+    gridProps['aria-rowcount'] = getItemCount(state.collection, (node) => node.type === 'item' || (node.type === 'loader' && node.props.isLoading));
```

Comment:
line 154 i think has to check this too since the loader is "tabbable"
also line 166?

Reply:
oh that right, didn't noticed since useSelectableCollection actually still ensures we have the right tabindex in this case. Relates to the last [comment](https://github.com/adobe/react-spectrum/pull/8326#discussion_r2121764059) I made below, ugh

Reply:
@devongovett any initial opinions on this? We had discussed previously that checks like https://github.com/adobe/react-spectrum/blob/081172e2f64071fbcf0f86c67b2291c9388ec631/packages/%40react-stately/layout/src/ListLayout.ts#L258 were a bit weird and debated whether or not the loaders should be in the collection